.github/workflows/integration-test.yml

```diff
@@ -22,79 +29,47 @@ on:
         options:
           - dev
           - staging
+      run-v1:
+        description: 'Run tests against Firebolt DB v1'
+        required: true
+        default: true
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+      run-v2:
+        description: 'Run tests against Firebolt DB v2'
+        required: true
+        default: true
+        type: choice
+        options:
+          - 'true'
+          - 'false'
```

Comment:
You can add `workflow_dispatch` section to V1 and V2 workflows to make them individually callable alongside the shared workflow. This way you don't need to have those input parameters.